.travis_script.sh

```diff
@@ -17,6 +17,7 @@ then
     exit $conf
 fi
 export PYTHONPATH=$(pwd)/tools/pylib/:$PYTHONPATH
+export MAKEFLAGS="-j 2"
 time make || exit
 time make check-unit-tests || exit
```

Comment:
I wasn't sure if it would pass the `-j 2` onto the child `make` -- not sure if the child inherits the parent environment or not.